binary_search_networks/binary_search_parser.py

```diff
@@ -20,9 +20,20 @@ def binary_search_parser():
 	parser.add_argument('--delta', default=50, type=int, help='The spacing between ni and nj when calculating the gradient of accuracy.')
 	parser.add_argument('--early_stopping_patience', default=3, type=int, help='Number of epochs with no improvement after which training will be stopped if early stopping is enabled.')
 	parser.add_argument('--epoch', default=150, type=int, help='The number of epochs to train for.')
+	parser.add_argument('--n', default=1000, type=int, help='The dimension of the hidden layer.')
```

Comment:
Good catch here, the PR #11 must have removed all the arguments that we need from the code. This must have escaped from my eyes. My apologies.